ci-cd/evaluate.py

```diff
@@ -15,49 +16,49 @@
 import torchvision.transforms as T
 from PIL import Image
 
-# (neptune) Fetch project
-project = neptune.get_project(
+# (Neptune) Set environment variable
+os.environ["NEPTUNE_PROJECT"] = "common/project-rl"
+
+# (Neptune) Fetch project
+project = neptune.init_project(
     api_token=os.getenv("NEPTUNE_API_TOKEN"),
-    name="common/project-rl",
 )
 
-# (neptune) Find latest run
+# (Neptune) Find latest run
 runs_table_df = project.fetch_runs_table().to_pandas()
 runs_table_df = runs_table_df.sort_values(by="sys/creation_time", ascending=False)
 run_id = runs_table_df["sys/id"].values[0]
 
-# (neptune) Resume run
-run = neptune.init(
+# (Neptune) Resume run
+run = neptune.init_run(
     api_token=os.getenv("NEPTUNE_API_TOKEN"),
-    project="common/project-rl",
-    run=run_id,
+    with_id=run_id,
     monitoring_namespace="evaluation/monitoring",
 )
 
-# (neptune) Download agent
-run["agent/policy_net"].download("policy_net.pth")
+# (Neptune) Initialize model version created by the run
+model_version = neptune.init_model_version(with_id=run["training/model/id"].fetch())
+
+# (Neptune) Download agent from model repository
+model_version["weights"].download("policy_net.pth")
 
-# (neptune) Fetch environment name, random seed number of actions in the env
+# (Neptune) Fetch environment name and number of actions in the env
 env_name = run["training/env_name"].fetch()
-rnd_seed = run["training/parameters/seed"].fetch()
-n_actions = run["agent/n_actions"].fetch()
+n_actions = model_version["n_actions"].fetch()
 
-# (neptune) Set number of episodes for evaluation and log in under separate namespace dedicated to evaluation
+# (Neptune) Set number of episodes for evaluation and log in under separate namespace dedicated to evaluation
 eval_episodes = 5
 run["evaluation/n_episodes"] = eval_episodes
 
-# (neptune) Upload evaluation script as separate file
-run["evaluation/script"].upload("ci-cd/evaluate.py")
+# (Neptune) Upload evaluation script as separate file
+run["evaluation/script"].upload(sys.argv[0])
```

Comment:
I prefer `"ci-cd/evaluate.py"` over `sys.argv[0]`. For someone without the knowledge that `sys.argv[0]` corresponds to the filename will seem confusing.

Reply:
I kept it as this so that it wouldn't need to be updated if the filename/path was changed.